desktop/lib/metaphysics.coffee

```diff
@@ -3,6 +3,7 @@ qs = require 'qs'
 request = require 'superagent'
 { extend, some } = require 'underscore'
 { METAPHYSICS_ENDPOINT } = require('sharify').data
+newrelic = require 'newrelic'
```

Comment:
I'm leveraging the fact that we have this b/c of https://github.com/artsy/artsy-newrelic